src/member/member.ts

```diff
@@ -9,6 +9,30 @@ export type MemberStorage = {
   maximum: number;
 };
 
+export type MemberStorageItem = {
+  id: UUID;
+  name: string;
+  size: number;
+  updatedAt: string;
+  path: string;
+  parent?: {
+    id: UUID;
+    name: string;
+  };
+};
+
+export type Pagination = {
+  totalItems: number;
+  totalPages: number;
+  page: number;
```

Comment:
From the specification, should it be `currentPage` ?
```suggestion
  currentPage: number;
```

Reply:
Yes, you're right